kubernetes/model/int_or_string.c

```diff
@@ -0,0 +1,60 @@
+#include <stdlib.h>
+#include <string.h>
+#include <stdio.h>
+#include "int_or_string.h"
+
+int_or_string_t *int_or_string_create()
+{
+    int_or_string_t *ios = calloc(sizeof(int_or_string_t), 1);
+    if (!ios) {
+        return NULL;
+    }
+    ios->type = IOS_DATA_TYPE_INT;
```

Comment:
it's not required, but probably slightly more friendly to set `ios->s = NULL`

Reply:
Thank you @brendandburns. It's done.